core/lib/multistore.ts

```diff
@@ -109,9 +109,9 @@ export function localizedRoute (routeObj: Route | string, storeCode: string) {
 }
 
 export function setupMultistoreRoutes (config, router, routes) {
-  if (config.storeViews.mapStoreUrlsFor.length > 0 && config.storeViews.multistore === true) {
+  if (config.storeViews.mapStoreUrlsFor.length > 1 && config.storeViews.multistore === true) {
     for (let storeCode of config.storeViews.mapStoreUrlsFor) {
-      if (storeCode) {
+      if (storeCode || (config.defaultStoreCode !== storeCode)) {
```

Comment:
Shouldn't it be `&&` instead of `||`?  The `or` condition makes only sence in case `storeCode` is null either empty which probably never occurs in this context
```suggestion
      if (storeCode && (config.defaultStoreCode !== storeCode)) {
```

Reply:
Yps, it's should be `&&`, I put `||` just for test.